js/src/lib/Search/index.js

```diff
@@ -43,6 +43,7 @@ const Search = props => (
       ]}
     />
     <SearchBox
+      autoFocus={window.location.pathname.includes('/packages/')}
```

Comment:
should be `window.location.pathname.includes('/package'), because it doesn't include the trailing slash :smile:

you can see it live at https://deploy-preview-417--yarnpkg.netlify.com/en/packages